CodeEdit/Features/CodeFile/CodeFileView.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 import SwiftUI
+import CodeEditSourceEditor
 import CodeEditTextView
```

Comment:
Redundant import?

Reply:
CEUndoManager is in the CETV package so both imports are needed.